src/js/view/floating/floating-controller.ts

```diff
@@ -220,6 +223,11 @@ export default class FloatingController {
                 transition: null,
                 'transition-timing-function': null
             });
+            setTimeout(() => {
+                this.transitionFloating(false);
+                // Perform resize and trigger "float" event responsively to prevent layout thrashing
+                this._model.trigger('forceResponsiveListener', {});
+            }, 50);
         };
 
         if (mobileFloatIntoPlace) {
```

Comment:
The new bug was not environment spefic (iOS/Chrome) is was most likely `mobileFloatIntoPlace` specific or some other mobile flag we have tossed around in here.

You might want to see why iOS Safari and iOS Chrome do not go down the same path. It's either a UA issue or a bounds issue in the page we're using to test.